controller/lib/core/sensors.h

```diff
@@ -22,13 +22,27 @@ Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
 #ifndef SENSORS_H
 #define SENSORS_H
 
-/*
- * @brief enum for the three different pressure sensors in the ventilator system
- */
-enum class pressureSensors {
-  PATIENT = 0, // patient pressure sensor
-  INHALATION,  // inhalation diff pressure sensor
-  EXHALATION   // exhalation diff pressure sensor
+#include "hal.h"
+
+class PressureSensors {
+public:
+  // Patient pressure sensor
+  static constexpr AnalogPinId PATIENT_PIN = AnalogPinId::HAL_A0;
+  // Inhalation diff pressure sensor
+  static constexpr AnalogPinId INHALATION_PIN = AnalogPinId::HAL_A1;
+  // Exhalation diff pressure sensor
+  static constexpr AnalogPinId EXHALATION_PIN = AnalogPinId::HAL_A2;
+
+  // min/max possible reading from MPXV5004GP [kPa]
+  static constexpr float P_VAL_MIN = 0.0f;
+  static constexpr float P_VAL_MAX = 3.92f;
+
+  // min/max possible reading from MPXV7002DP [kPa]
+  static constexpr float DP_VAL_MIN = -2.0f;
+  static constexpr float DP_VAL_MAX = 2.0f;
```

Comment:
Unless and until these are inline variables, they need storage in the cpp file, e.g.

```
// sensors.cpp
AnalogPinId PressureSensors::PATIENT_PIN;
```

I think the way you  can observe this fail is if you "ODR-use" the variable, e.g. by taking its address.

Reply:
Changed to inline as well.